web/src/user/LibraryPage/LibraryPageImpl.ts

```diff
@@ -125,7 +130,7 @@ export class LibraryPage extends AKElement {
     }
 
     renderSearch() {
-        return html`<ak-library-list-search .apps="{this.apps.results}"></ak-library-list-search>`;
+        return html`<ak-library-list-search .apps=${this.apps.results}></ak-library-list-search>`;
```

Comment:
This was the real issue; everything else is just dealing with errors to the console when the field is cleared (when there is no `app[0]`).  I got this to work, then moved it here and effed-up my argument call.  Massive apologies.